rh15d/readatmos.c

```diff
@@ -226,12 +294,19 @@ void depth_refine(Atmosphere *atmos, Geometry *geometry, double Tmax) {
   for (k = k0+1; k <= k1; k++) {
     tdiv = fabs(log10(atmos->T[k]) - log10(atmos->T[k-1]))/lg1;
     /* rho is not available, so nH[0] used instead */
-    rdiv = fabs(log10(atmos->nH[0][k]) - log10(atmos->nH[0][k-1]))/lg1;
+    rdiv   = fabs(log10(atmos->nH[0][k]) - log10(atmos->nH[0][k-1]))/lg1;
     taudiv = fabs(log10(tau[k]) - log10(tau[k-1]))/0.1;
-    aind[k] = aind[k-1] + MAX(MAX(tdiv,rdiv),taudiv);
+    vdiv   = fabs(geometry->vel[k] - geometry->vel[k-1]) * vel_scale; // JdlCRL: added Vdiv
```

Comment:
Please do not add additional whitespace before the `=` to align the operator. See [PEP 8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).